*** glibc.spec	2016-02-16 15:31:16.000000000 +0000
--- glibc.spec.new	2016-03-28 20:31:27.526094339 +0000
***************
*** 730,735 ****
--- 730,737 ----
  # Upstream BZ 18557: Fix ruserok scalability issues.
  Patch2066: glibc-rh1216246.patch
  
+ Patch10000: glibc-2.17-crypt-bcrypt.patch
+ 
  ##############################################################################
  # End of glibc patches.
  ##############################################################################
***************
*** 1320,1325 ****
--- 1322,1328 ----
  %patch1618 -p1
  %patch1619 -p1
  %patch1620 -p1
+ %patch10000 -p1
  
  ##############################################################################
  # %%prep - Additional prep required...
***************
*** 1348,1353 ****
--- 1351,1360 ----
  # Ensure *-kw.h files are current to prevent regenerating them.
  touch locale/programs/*-kw.h
  
+ # Patch bcrypt
+ mv crypt/crypt.h crypt/gnu-crypt.h
+ cp -f $RPM_BUILD_DIR/../tmp/crypt_blowfish-1.3/{*.h,*.c,x86.S} ./crypt/
+ 
  ##############################################################################
  # Build glibc...
  ##############################################################################
